### PR TITLE
samples: bluetooth: peripheral_uart: add support for 54l15

### PR DESCRIPTION
**Pull request by Pieloaf**
Add support to bt peripheral uart sample for 54l15
Signed-off-by: Pierce Lowe <pierce.lowe@nordicsemi.no>

### CODE REVIEW

**Review by olivier-le-sage — APPROVED** [review]

**Review by jori-nordic — APPROVED** [review]

File: samples/bluetooth/peripheral_uart/boards/nrf54l15pdk_nrf54l15_cpuapp.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
```
Comment (maje-emb) [inline samples/bluetooth/peripheral_uart/boards/nrf54l15pdk_nrf54l15_cpuapp.conf:2]:
```suggestion
# Copyright (c) 2024 Nordic Semiconductor
```

File: samples/bluetooth/peripheral_uart/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
```
Comment (maje-emb) [inline samples/bluetooth/peripheral_uart/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay:2]:
```suggestion
 * Copyright (c) 2024 Nordic Semiconductor ASA
```

**Review by maje-emb — COMMENTED** [review]
Shouldn't this commit be included?
https://github.com/nrfconnect/sdk-nrf-next/commit/3914fedb63dd083842421c4c7f82ab3460297818

**Comment by grochu** [comment]
> Shouldn't this commit be included? [nrfconnect/sdk-nrf-next@3914fed](https://github.com/nrfconnect/sdk-nrf-next/commit/3914fedb63dd083842421c4c7f82ab3460297818)

Hi @Pieloaf, if you want to migrate nRF54 support in this sample to NCS, please make sure all changes from nrf-sdk-next (that diverged from sdk-nrf) are upstreamed to NCS.

**Comment by rakons** [comment]
Please note this PR: https://github.com/nrfconnect/sdk-nrf/pull/14490. It is provided because we wish to use this same module for peripheral_uart and central_uart samples, not to copy it.

**Review by alwa-nordic — APPROVED** [review]

**Pull request merged**